packages/bull-vault/src/AuctionBull.sol

```diff
@@ -377,7 +382,7 @@ contract AuctionBull is UniFlash, Ownable, EIP712 {
                         require(currentPrice >= prevPrice, "AB7");
                     }
                     prevPrice = currentPrice;
-
+                    // transfer from order to contract
```

Comment:
we should fire events for traded orders probably

Reply:
(both buy and sell)

Reply:
added